akka-docs/rst/java/persistence.rst

```diff
@@ -326,6 +326,12 @@ In this case no stashing is happening, yet events are still persisted and callba
 While it is possible to nest mixed ``persist`` and ``persistAsync`` with keeping their respective semantics
 it is not a recommended practice, as it may lead to overly complex nesting.
 
+.. warning::
+  While it is possible to next ``persist`` calls within one another, 
+  it is *not* legal to nest ``persist`` calls in Futures! Doing so will break the 
```

Comment:
not really about nesting, it's simply not legal to call `persist` from a Future callback or any other thread than the actor's message processing thread